ibm_catalog.json

```diff
@@ -0,0 +1,146 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "description": "Supports multifactor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA.",
+                    "title": "Multifactor authentication"
+                },
+                {
+                    "description": "Restrict API key creation (on-off)\nRestrict service ID creation (on-off)",
+                    "title": "Restrict API Key and Service IDs"
+                },
+                {
+                    "description": "Session activity timeout (seconds)\nSession inactivity timeout (seconds)",
+                    "title": "Session timeouts"
+                },
+                {
+                    "description": "Access token expiration (seconds)\nRefresh token expiration (seconds)",
+                    "title": "Token Expiration"
+                },
+                {
+                    "description": "off/Any Allowed - on/Allow only specified IP subnets or IP addresses. \n\nTwo control modes are supported Monitor \u0026 Restrict",
+                    "title": "Restrict IP address access"
+                },
+                {
+                    "description": "Global shell settings (on-off)\nPublic access group (on-off)",
+                    "title": "Global shell settings and public access group"
+                }
```

Comment:
Check with @SirSpidey but do we want to duplicate features at the product leve that we already have at the flavor level? Or should the product level just have a higher level summary?

Reply:
Right now there is only one variation. Hence the product-level features should be the same as the flavor-level features. The reasoning behind duplicating the features is no matter where the user is exploring at the product or flavor level - the user will see all the features. 

I can split up the features at both levels if that's the requirement.

Reply:
We don't want the same information in both places because it shows up on the same page in the catalog (see screenshot).

The screenshot shows what we have for the flavor/variation level for VPC landing zone. Code is at https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/ibm_catalog.json#L137-L176. So it looks like, to be consistent, we should show information that is different from the product level.

<img width="506" alt="image" src="https://user-images.githubusercontent.com/8194208/235191740-ba2b4aee-7917-4d69-81ac-54f3cce82dec.png">